elements/pfe-switch/src/pfe-switch.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Switch",
+  "description": "Creates a fancy switch element from checkbox markup",
+  "type": "object",
+  "tag": "pfe-switch",
+  "class": "PfeSwitch",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "default": {
+          "title": "Default slot",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "raw"
+              }
+            ]
+          }
+        }
+      },
+      "required": ["default"]
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "form": {
```

Comment:
The attributes section should list out all the data attributes supported by your component: `message-on`, `message-off`, `hide-label`.  The form indication could go in your slot array above instead of referencing a raw field, you could indicate form to tell the component that it only accepts a form element in the slot.

Reply:
Is there an example or existing component I can code-monkey off of?